install/kubernetes/quick-install.yaml

```diff
@@ -317,6 +317,12 @@ rules:
   - ciliumidentities/status
   verbs:
   - '*'
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - get
```

Comment:
Based on the suggestions we would need to add `list` and `watch` verbs here.

Reply:
Done